test/test_tensorflow.py

```diff
@@ -383,6 +383,114 @@ def test_horovod_allgather(self):
                             tf.equal(tf.cast(rank_tensor, tf.int32), value))),
                         "hvd.allgather produces incorrect gathered tensor")
 
+    def test_horovod_allgather_fused(self):
+        """Test that the allgather correctly gathers 1D, 2D, 3D tensors
+        with Tensor Fusion."""
+        hvd.init()
+        rank = hvd.rank()
+        size = hvd.size()
+
+        with self.test_session(config=self.config) as session:
```

Comment:
Sorry, can you fix tests as well?  With #670 there's no `self.config` anymore, just `config`.